src/java/org/apache/cassandra/net/SensorsCustomParams.java

```diff
@@ -154,20 +186,49 @@ public static <T> void addReadSensorToResponse(Message.Builder<T> response, Requ
                             (ignored) -> SensorsCustomParams.READ_BYTES_TABLE);
     }
 
+    /**
+     * A utility method to encode internode bytes sensor values in the internode response message.
+     */
+    public static <T> void addInternodeBytesSensorToResponse(Message.Builder<T> response, RequestSensors sensors, Context context)
+    {
+        addSensorToResponse(response, sensors, context, Type.INTERNODE_BYTES,
+                            (sensor) -> SensorsCustomParams.encodeTableInInternodeBytesRequestParam(sensor.getContext().getTable()),
+                            (sensor) -> SensorsCustomParams.encodeTableInInternodeBytesTableParam(sensor.getContext().getTable()));
+    }
+
+    /**
+     * A utility method to encode internode bytes sensor values in the internode response message. This method will iterate
+     * over all the write sensors in the {@link RequestSensors} and encode the sensor values in the internode response message.
+     */
+    public static <T> void addInternodeBytesSensorToResponse(Message.Builder<T> response, RequestSensors sensors)
```

Comment:
Consider renaming because we have the same name in the function above and they do different things. Proposal: `addInternodeBytesFromRequestSensors`

Reply:
I took a pass on this class to simplify it and improve method names, hard to do without some refactor, but it is minimal and captured in this commit: https://github.com/datastax/cassandra/pull/1152/commits/910f3a6e134f1601cae981d7f5787bd2d760c3fa